docs/user-guide/getting-started.md

```diff
@@ -13,6 +13,8 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 
 ## Installation and Usage
 
+Prerequisites: [Node.js](https://nodejs.org/en/) (>=4.x), npm version 3+.
```

Comment:
do we support npm v2?

Reply:
Yes, we support the default version of npm that comes with Node 4, which is npm 2.